package.json

```diff
@@ -41,7 +41,8 @@
     "iron-session": "~6.3.1",
     "jose": "~5.6.3",
     "leb": "^1.0.0",
-    "pluralize": "8.0.0"
+    "pluralize": "8.0.0",
+    "qs": "6.14.0"
```

Comment:
I'd avoid adding the new package `qs` here if it's possible to use the native `URLSearchParams` (to avoid extra packages installed in the SDK).